RecoLocalCalo/EcalRecAlgos/src/PulseChiSqSNNLS.cc

```diff
@@ -311,10 +311,12 @@ bool PulseChiSqSNNLS::NNLS() {
       const unsigned int nActive = npulse - _nP;
       
       updatework = aTbvec - aTamat*_ampvec;      
+      Index idxwmaxprev = idxwmax;
+      double wmaxprev = wmax;
```

Comment:
These two *prev variables should be made const to ensure the compiler optimizes them out.